indexer/src/main/java/org/uniprot/store/indexer/literature/LiteratureJob.java

```diff
@@ -27,7 +27,8 @@ public Job indexLiteratureSupportingData(
             JobExecutionListener jobListener) {
         return this.jobs
                 .get(Constants.LITERATURE_LOAD_JOB_NAME)
-                // .start(literatureStatistics) // DO NOT MERGE TO MASTER COVID-19 CODE
+                //                .start(literatureStatistics) // DO NOT MERGE TO MASTER COVID-19
```

Comment:
I think you need to uncomment this line. We might have merged it to master by mistake...